src/borg/helpers/parseformat.py

```diff
@@ -843,6 +845,14 @@ def prepare_hash_function(self, hash_function):
             hash = self.xxh64()
         return hash
 
+    def hash_chunker_params(self, hash_function, item):
+        hash = self.prepare_hash_function(hash_function)
+
+        chunker_params = self.archive.metadata.get('chunker_params')
+        for info in chunker_params:
+            hash.update(bytes(info))
+        return hash.hexdigest()
+
```

Comment:
and as i said, one can just show the chunker_params and not compute a hash for them.

Reply:
done.